BUILD.bazel

```diff
@@ -1486,6 +1486,7 @@ ray_cc_test(
     tags = [
         "stats",
         "team:core",
+        "no_tsan",
```

Comment:
Q: this will keep the status quo right? (we will still run tests with tsan)

Reply:
Yess, it's the same behavior as before